src/utils/fetchOpenSaucedApiData.ts

```diff
@@ -1,9 +1,8 @@
-export const getOpenSaucedUser = async (username: string) => {
-  const response = await fetch(
-    `https://api.opensauced.pizza/v1/users/${username}`
+export const getOpenSaucedUser = (username: string) => {
+  fetch(`https://api.opensauced.pizza/v1/users/${username}`).then(
+    (response) => {
+      if (response.status !== 200) return false;
+    }
   );
-  if (response.status !== 200) {
-    return null;
-  }
-  return await response.json();
+  return true;
 };
```

Comment:
refactor (non-blocking): There is no need to do an explicit return of `false` or `true`. Question, how do we plan on handling errors, or is it good enough to see if we have a `200`?

```diff
- export const getOpenSaucedUser = (username: string) => {
+ export const getOpenSaucedUser = async (username: string) => {
-  fetch(`https://api.opensauced.pizza/v1/users/${username}`).then(
+  const response = await fetch(`https://api.opensauced.pizza/v1/users/${username}`)
-    (response) => {
-      if (response.status !== 200) return false;
-    }
-  );

-  return true;
+. return response.status === 200;
};
```

Reply:
At the moment a check for 200 seems to suffice. I've added a catch block just in case.
c9fef9068be49fdf70f6fe0af2b8c0b3a87d5c6a